### DISCUSSION TITLE
Column Width Problem

### DISCUSSION THREAD

**Discussion by sksbell** [LesFerch/WinSetView#108] [Q&A] [answered]
New user here - trying to set column widths in details view and the widths never change - have to do the annoying right click "size all columns to width" every time.  I've tried both pixels and ems.  For the later, used values less than the default 34.  What am I doing wrong?  (I have checked and the ems values are saved, and I did submit.)

Thanks!
SKSBELL

**Comment by LesFerch** [comment]
Setting custom widths should just work. That's something I use all the time. Please share your **Win10.ini** file from the WinSetView AppData folder. I'm assuming you're using Windows 10 or Windows 11. Are you using anything that could be interfering, such as [WindowManager](https://www.desksoft.com/WindowManager.htm)?

**Reply by sksbell** [reply]
Thanks for the speedy reply!  Running Windows 11 home.

Not using WindowManager.  Don't know enough to know what else would interfere...
[SKSWin10.txt](https://github.com/LesFerch/WinSetView/files/14897747/SKSWin10.txt)

**Comment by LesFerch** [comment]
Your settings work perfectly fine for me. I get the narrower Name column, the wider Size column, etc.

Look through your installed programs for anything that affects what you see in Explorer. Another possibility is that you're looking at folders that have there own special view, such as [box](https://www.box.com/) folders.

Please show a screenshot of a folder that doesn't honor the widths you've set. That might provide some clues.

**Comment by sksbell** [comment]
I uninstalled Agent Ransack, but didn't see anything else that looked suspect. Screenshot  of a documents folder attached.
![wide columns](https://github.com/LesFerch/WinSetView/assets/166261465/d8d8e005-2b1c-4e34-9767-20f14547f2d7)

**Comment by LesFerch** [comment]
You've got Box and Dropbox installed. Folders controlled by that software have their own special views with no defaults that can be set via the registry. The folder you're showing looks "normal" in the tab (C:\Users\sksbr\Documents\00...), but the presence of "Start Backup" in the address bar indicates a special view.

Take a look at a folder that you're sure has nothing to do with Box or Dropbox. For example, how does C:\Windows\System32 look?

**Comment by sksbell** [comment]
The "start backup" is microsoft onedrive (which I don't really use) and I removed it and rebooted.  Folders still look the same, including  C:\Windows\System32.

**Comment by LesFerch** [comment]
Let's see how it looks under a different profile. Create a new local user account (make it an administrator), sign in with that account, run WinSetView with the same settings, and then check some folders and see what you get.

**Comment by sksbell** [comment]
It works under the other local user...
![Screenshot 2024-04-07 185423](https://github.com/LesFerch/WinSetView/assets/166261465/9d34168c-bd26-4516-a877-6cc937e66598)

And when I came back to my user account, it works on the thumb drive I stored the screenshot on, but not in  my documents.
![Screenshot 2024-04-07 190227](https://github.com/LesFerch/WinSetView/assets/166261465/b5291442-c312-4251-a56c-751104c7b6ba)

**Comment by LesFerch** [comment]
I suspect Box is causing the problem, although it shouldn't be causing a problem for a non-Box folder, such as System32. I don't know if you can disable or uninstall Box temporarily to see if that makes a difference, but that's all I can think of trying.

**Comment by sksbell** [comment, marked as answer]
I logged out of Box, and now it works!
